homeassistant/components/plant.py

```diff
@@ -97,6 +104,16 @@ def async_setup(hass, config):
         async_track_state_change(hass, sensor_entity_ids, entity.state_changed)
         entities.append(entity)
 
+        if CONF_GROUP_NAME in plant_config and \
+                plant_config[CONF_GROUP_NAME] is not None:
+            group_name = plant_config[CONF_GROUP_NAME]
+            members = ['{}.{}'.format(DOMAIN, plant_name)]  # own entity id
+            members.extend(sensor_entity_ids)
+            hass.states.async_set('group.{}'.format(group_name), 'off', {
```

Comment:
Use the group service calls...

Reply:
Not sure what you mean with "service calls". I changed that part to the way it's done in ```components/demo.py```

Is that what you proposed?